mailpoet/tests/unit/Newsletter/Statistics/Export/StatisticsExporterTest.php

```
@@ -119,6 +119,67 @@ public function testBuildAggregateRowMatchesHeaderColumnCount() {
     verify(count($row))->equals(count($headers));
   }
 
+  public function testItExportsRecipientsFromFilterRows() {
+    $newsletter = $this->createNewsletter(99, 'Recipients', null, '2026-02-01 00:00:00');
+    $stats = $this->createStats(0, 0, 0, 0, 0, 0, null);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$stats'. (undefined)

(UnusedLocalVariable)